evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -225,4 +228,17 @@
     {% endif %}
 </td>
 
+<script>
+    var exam_creation_modal = document.getElementById("exam_creation_modal");
+    var btn = document.querySelectorAll('button[evaluation_id="{{ evaluation.id }}"]')[0];
+    var span = document.getElementsByClassName("close")[0];
```

Comment:
btn is only available, if `not evaluation.has_exam`. This causes a javascript error if there are evaluations with exams visible